### PR TITLE
Fix apt package lookups in install_deps.pl

### PR DESCRIPTION
**Pull request by jeanpaulgalea**
Check commit message for details.

### CODE REVIEW

File: bin/install_deps.pl [hunk 1]
```diff
@@ -451,10 +451,12 @@ sub name_overrides {
...
 # MacPorts ($dport), yum, and apt.
     my @modules = (
-        { module=>'LWP::UserAgent', info => { cat=>'www', port=>'p5-libwww', dport=>'p5-libwww-perl', yum=>'perl-libwww-perl' }, },
-        { module=>'Mail::Send'    , info => { port => 'Mail::Tools', }  },
-        { module=>'Date::Parse'   , info => { port => 'TimeDate',    }  },
-        { module=>'LWP'           , info => { port => 'p5-libwww',   }  },
+        { module=>'LWP::UserAgent'    , info => { cat=>'www', port=>'p5-libwww', dport=>'p5-libwww-perl', yum=>'perl-libwww-perl' }, },
+        { module=>'Mail::Send'        , info => { port => 'Mail::Tools', }  },
+        { module=>'Date::Parse'       , info => { port => 'TimeDate',    }  },
+        { module=>'LWP'               , info => { port => 'p5-libwww',   }  },
```
Comment (msimerson) [inline bin/install_deps.pl]: Isn't this is redundant, being covered by LWP::UserAgent above?

Comment (jeanpaulgalea) [inline bin/install_deps.pl]: @msimerson there's only whitespace changes in those four lines, i.e. lines 454 to 457.

Comment (jeanpaulgalea) [inline bin/install_deps.pl]: @msimerson or you mean if LWP and LWP::UserAgent are installed by a single apt package? I think they're provided by `lib-www-perl`;

```
root@trusty:~# dpkg -S /usr/share/perl5/LWP
libwww-perl, liblwp-mediatypes-perl, liblwp-protocol-https-perl: /usr/share/perl5/LWP
```
```
root@trusty:~# dpkg -S /usr/share/perl5/LWP/UserAgent.pm 
libwww-perl: /usr/share/perl5/LWP/UserAgent.pm
```
```
root@trusty:~# dpkg -l | grep -i lwp
ii  liblwp-mediatypes-perl             6.02-1                           all          module to guess media type for a file or a URL
ii  liblwp-protocol-https-perl         6.04-2ubuntu0.1                  all          HTTPS driver for LWP::UserAgent
```
```
root@trusty:~# dpkg -L liblwp-protocol-https-perl 
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/liblwp-protocol-https-perl
/usr/share/doc/liblwp-protocol-https-perl/changelog.Debian.gz
/usr/share/doc/liblwp-protocol-https-perl/copyright
/usr/share/perl5
/usr/share/perl5/LWP
/usr/share/perl5/LWP/Protocol
/usr/share/perl5/LWP/Protocol/https.pm
/usr/share/man
/usr/share/man/man3
/usr/share/man/man3/LWP::Protocol::https.3pm.gz
```
```
root@trusty:~# dpkg -L liblwp-mediatypes-perl 
/.
/usr
/usr/share
/usr/share/perl5
/usr/share/perl5/LWP
/usr/share/perl5/LWP/MediaTypes.pm
/usr/share/perl5/LWP/media.types
/usr/share/man
/usr/share/man/man3
/usr/share/man/man3/LWP::MediaTypes.3pm.gz
/usr/share/doc
/usr/share/doc/liblwp-mediatypes-perl
/usr/share/doc/liblwp-mediatypes-perl/copyright
/usr/share/doc/liblwp-mediatypes-perl/changelog.Debian.gz
```

Comment (msimerson) [inline bin/install_deps.pl]: Ahh, I see. I was looking at it on a small screen and...never mind. %-}

**Comment by msimerson** [comment]
thanks!

**Comment by jeanpaulgalea** [comment]
Thanks for being a great upstream and for providing such a piece of
software in the first place!

**Pull request merged**